discord_bots/commands.py

```diff
@@ -1578,6 +1580,11 @@ async def createdbbackup(ctx: Context):
         colour=Colour.green(),
     )
 
+@bot.command(name= 'restart')
```

Comment:
```suggestion
@bot.command()
```
not needed if it matches function name